events.go

```diff
@@ -7,18 +7,22 @@ package godspeed
 import (
 	"bytes"
 	"fmt"
+	"strconv"
 	"strings"
 )
 
 var eventKeys = []string{"date_happened", "hostname", "aggregation_key", "priority", "source_type_name", "alert_type"}
 var eventMarkers = []rune{'d', 'h', 'k', 'p', 's', 't'}
 
+var escapeEventReplacer = strings.NewReplacer("\n", "\\n")
+var pipesReplacer = strings.NewReplacer("|", "")
```

Comment:
These should probably go in `shared.go`.

Reply:
makes sense, I'll move them there along with pipeReplace().